src/serializer/ResidualHeapSerializer.js

```diff
@@ -1666,6 +1679,10 @@ export class ResidualHeapSerializer {
       let ob = serializedArgs[0];
       invariant(ob !== undefined);
       return t.callExpression(this.preludeGenerator.memoizeReference("Object.assign"), [ob]);
+    } else if (val.kind === "template for prototype member expression") {
```

Comment:
@sebmarkbage I wasn't sure if this was right, but we hit this abstract value in the 2nd test I added.